src/plugins/UnsuppressEmbeds.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Vencord, a modification for Discord's desktop app
+ * Copyright (c) 2022 Vendicated and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+import { addContextMenuPatch, NavContextMenuPatchCallback, removeContextMenuPatch } from "@api/ContextMenu";
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+import { Menu, PermissionStore, RestAPI } from "@webpack/common";
+
+const EMBED_SUPPRESSED = 1 << 2;
+const MANAGE_MESSAGES = 1n << 13n;
+
+const messageContextMenuPatch: NavContextMenuPatchCallback = (children, props) => () => {
+    const { message: { embeds, flags } } = props;
```

Comment:
And this (forgot, sorry)
```suggestion
    const { message: { embeds, flags, author } } = props;
    const channel = ChannelStore.getChannel(props.channel.id);
```